spec/controllers/book_suggetions_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book_suggestion' do
+      let!(:book_suggestion) { create(:book_suggestion) }
```

Comment:
You don't need to create it if you don't plan to use the saved record. You can use `build(:book_suggestion)` to instantiate the model without saving it.
Also, you can omit the use of `!` un the let, in this case you don't need to execute the let before any example